### PR TITLE
Mail2kvanitha/c02-network08

### PR DESCRIPTION
**Pull request by mail2kvanitha**
# Description

Exercise (e.g. C01-E01): c02-network08

Student Name: Vanitha Kaliyaperumal

> A brief explanation of what has been done.
Added NACL rule to block a network to access a subnet and also added NACL rule to block traffic to particular external network.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network08/mail2kvanitha/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Commands Execution Output
+
+- Answer the following questions
+  - What is the difference between inbound x outbound?
+    ```
+    Inbound Rules are for traffic incoming to the subnet to which the NACLs are attached. 
+    Outbound Rules are for the traffic leaving the subnet to which the NACLs are attached. i
+    By default, the default NACLs  allow ALL inbound and outbound traffic. 
+    But for custom NACLs(devopsacademy-nacls),by default ALL inbound and outbound traffic is denied till RULES are added.
```
Comment (denstorti) [inline classes/02class/exercises/c02-network08/mail2kvanitha/README.md:11]: 👍

File: classes/02class/exercises/c02-network08/mail2kvanitha/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    ```
+    
+  - What is going to happen by default? Everything allowed or denied?
+    ```
+    By default, the default NACLs allow ALL inbound and outbound traffic to/from the subnet to which it is attached.
```
Comment (denstorti) [inline classes/02class/exercises/c02-network08/mail2kvanitha/README.md:16]: the question is related to the table in the exercise description.

File: classes/02class/exercises/c02-network08/mail2kvanitha/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      {
+                    "CidrBlock": "172.16.200.0/24",
+                    "Egress": false,
+                    "PortRange": {
+                        "From": 22,
+                        "To": 22
+                    },
+                    "Protocol": "6",
+                    "RuleAction": "deny",
+                    "RuleNumber": 102
```
Comment (denstorti) [inline classes/02class/exercises/c02-network08/mail2kvanitha/README.md:37]: Is this the correct Rule number?

Remember that NACLs are evaluated from the lower rule number to the higher.

https://docs.aws.amazon.com/vpc/latest/userguide/vpc-network-acls.html

File: classes/02class/exercises/c02-network08/mail2kvanitha/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    > aws ec2 create-network-acl-entry --cidr-block 180.200.50.0/24  --egress --network-acl-id acl-087151bf5955637aa --protocol "-1" --rule-action deny --rule-number 103 
+
+    > aws ec2 describe-network-acls 
+
+           {
+                    "CidrBlock": "180.200.50.0/24",
+                    "Egress": true,
+                    "Protocol": "-1",
+                    "RuleAction": "deny",
+                    "RuleNumber": 103
```
Comment (denstorti) [inline classes/02class/exercises/c02-network08/mail2kvanitha/README.md:54]: Same comment as above.
Remember that NACLs are evaluated from the lower rule number to the higher.

https://docs.aws.amazon.com/vpc/latest/userguide/vpc-network-acls.html

File: classes/02class/exercises/c02-network08/mail2kvanitha/README.md (hunk 2 above)
Comment (mail2kvanitha) [reply classes/02class/exercises/c02-network08/mail2kvanitha/README.md:16]: thanks, got it now. Updated the answer accordingly.

**Pull request merged**